pkg/controller/lifecycle/actuator.go

```diff
@@ -282,6 +310,55 @@ func (a *actuator) getShootAgentResources(defaultPeriod time.Duration, pingEnabl
 	return shootResources, nil
 }
 
+func buildAgentNetworkPolicy() client.Object {
+	tcp := corev1.ProtocolTCP
+	podGRPCPort := intstr.FromInt(common.PodNetPodGRPCPort)
+	podHttpPort := intstr.FromInt(common.PodNetPodHttpPort)
+	hostGRPCPort := intstr.FromInt(common.HostNetPodGRPCPort)
```

Comment:
Is there not a port missing? I would have expected `hostHttpPort` to be here as well.

Reply:
added hostHttpPort